server/runtime/pom.xml

```diff
@@ -598,5 +598,41 @@
             </dependency>
          </dependencies>
       </profile>
+      <profile>
+         <id>image</id>
+         <build>
+            <plugins>
+               <plugin>
+                  <groupId>io.fabric8</groupId>
+                  <artifactId>docker-maven-plugin</artifactId>
+                  <version>0.45.0</version>
```

Comment:
Should we define the plugin in `pluginManagement` as part of `build/configuration/pom.xml` and use a version property to be consistent?